dex/testing/loadbot/loadbot.go

```diff
@@ -698,3 +715,28 @@ func loadNodeConfig(symbol, node string) map[string]string {
 	}
 	return cfg
 }
+
+func symmetricWalletConfig(numCoins int, midGap uint64) (
+	minBaseQty, maxBaseQty, minQuoteQty, maxQuoteQty uint64) {
+
+	minBaseQty = uint64(maxOrderLots) * uint64(numCoins) * lotSize
+	minQuoteQty = calc.BaseToQuote(midGap, minBaseQty)
+	// Ensure enough for registration fees.
+	if minBaseQty < 2e8 {
+		minBaseQty = 2e8
```

Comment:
I know you're just moving this function, but why is it not just `minBaseQty += 2e8` unconditionally?

Reply:
Not sure, lets do that.